src/Umbraco.Cms.Api.Management/Controllers/Document/Versions/DocumentVersionControllerBase.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Management.Routing;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Core.Services.OperationStatus;
+using Umbraco.Cms.Web.Common.Authorization;
+
+namespace Umbraco.Cms.Api.Management.Controllers.Document.Versions;
+
+[ApiController]
+[VersionedApiBackOfficeRoute($"{Constants.UdiEntityType.Document}-version")]
+[ApiExplorerSettings(GroupName = $"{Constants.UdiEntityType.Document} Version")]
+[Authorize(Policy = AuthorizationPolicies.TreeAccessDocuments)]
+public class DocumentVersionControllerBase : ManagementApiControllerBase
+{
+    protected readonly IContentVersionService ContentVersionService;
```

Comment:
```suggestion
    private readonly IContentVersionService _contentVersionService;
```

Reply:
Need it to be protected to be used by inheriting classes